packages/mdc-icon-button/README.md

```diff
@@ -0,0 +1,170 @@
+<!--docs:
+title: "Icon Buttons"
+layout: detail
+section: components
+iconId: button
+path: /catalog/buttons/icon-buttons/
+-->
+
+# Icon Buttons
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material.io/design/components/buttons.html#toggle-button">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/icon-toggles.png" width="20" alt="Icon buttons screenshot">
+  </a>
+</div>-->
+
+Icon buttons allow users to take actions, and make choices, with a single tap.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/design/components/buttons.html#toggle-button">Material Design guidelines: Toggle buttons</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components.github.io/material-components-web-catalog/#/component/icon-toggle">Demo</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install @material/icon-button
+```
+
+## Usage
+
+### HTML Structure
+
+```html
+<button class="mdc-icon-button material-icons">favorite</button>
+```
+
+> Note: The MDC Icon Button can be used with `<button>` and `<a>` tags.
+
+> Note: IE11 will not center the icon properly if there is a newline or space after the material icon text.
+
+### Styles
+
+```scss
+@import "@material/icon-button/mdc-icon-button";
+```
+### JavaScript Instantiation
+
+The icon button will work without JavaScript, but you can enhance it to have a ripple effect by instantiating `MDCRipple` on the root element. 
+See [MDC Ripple](../mdc-ripple) for details.
+
+```js
+import {MDCRipple} from '@material/ripple';
+
+const iconButtonRipple = new MDCRipple(document.querySelector('.mdc-icon-button'));
```

Comment:
Should we include another line here for `iconButtonRipple.unbounded = true` in order to get the ripple to look correct?